--- ibmswtpm2-1661/src/TpmToOsslMath.h.orig	2021-01-23 00:14:01.000000000 +0100
+++ ibmswtpm2-1661/src/TpmToOsslMath.h	2022-07-01 22:24:21.822533492 +0200
@@ -73,7 +73,7 @@
 
 #define SYMMETRIC_ALIGNMENT RADIX_BYTES
 
-#if OPENSSL_VERSION_NUMBER >= 0x10200000L
+#if OPENSSL_VERSION_NUMBER >= 0x30100000L
 // Check the bignum_st definition in crypto/bn/bn_lcl.h and either update the
 // version check or provide the new definition for this version.
 #   error Untested OpenSSL version
